src/app/record/pages/my-orcid/my-orcid.component.ts

```diff
@@ -124,6 +138,9 @@ export class MyOrcidComponent implements OnInit, OnDestroy {
             this._robotsMeta.disallowRobots()
           }
           this._openGraph.addOpenGraphData(userRecord, { force: true })
+          if (environment.APPCUES) {
+            this.initializeAppCues(this.userInfo, this.userRecord)
```

Comment:
Should this be wrapped with an `if`, that only calls the `initializeAppCues` once? 

At the moment this will be called every time an endpoint responds, which results in calling the app cues `identity` function multiple times with potentially different values. 

What do you think? 🤔

Reply:
done!